README.rst

```diff
@@ -84,7 +84,9 @@ Using the API
     from google.cloud import logging_v2
 
     client = logging_v2.LoggingServiceV2Client()
-    entries = []
+    e = logging_v2.types.LogEntry(
+        text_payload="text")
```

Comment:
please update "Text" to something more meaningful, like "this is a log statement"

Reply:
I used the exact phrase :)